webapp/src/app.tsx

```diff
@@ -44,6 +44,7 @@ import * as blocks from "./blocks"
 import * as serialindicator from "./serialindicator"
 import * as draganddrop from "./draganddrop";
 import * as notification from "./notification";
+import { initElectron, isElectron, isPxtElectron } from "./electron";
```

Comment:
I find importing variables in the global namespace messy. Can we do electron.initElectron and so forth?

Reply:
ah sure, it's the new VS Code autoimport feature that does that